### DISCUSSION TITLE
Event name translation problem

### DISCUSSION THREAD

**Discussion by Terracotta-62** [artisan-roaster-scope/artisan#1216] [General]
![e713f985ff3259f59583999756d08dd](https://github.com/artisan-roaster-scope/artisan/assets/116416332/7ce4d71e-4118-461f-a3e3-87b36fa234d4)

**Comment by MAKOMO** [comment]
Very good find! It turned out that on loading a background profile the encoded event names were not correctly decoded. Just fixed.
